src/component/App.jsx

```diff
@@ -120,9 +121,11 @@ function App() {
                     <RecipeFocusStart />
                   </Route>
                   <Route exact path="/recipe/:recipeId/reviews">
-                    Reviews
                     <ReviewList />
                   </Route>
+                  <Route exact path="/recipe/:recipeId/review/:reviewId">
```

Comment:
should the end of this path be recipes/:recipeId/reviews/:reviewId?
Probably not necessary, but might be more clear